backend/src/main/java/org/sefglobal/core/controller/MultiverseController.java

```diff
@@ -80,9 +89,30 @@ public Ambassador createAmbassador(@PathVariable long id,
         }).orElseThrow(() -> new ResourceNotFoundException("University not found"));
     }
 
-    //todo: Implement an endpoint to list all ambassadors in a university
-    //todo: Implement an endpoint to delete ambassador (change active status)
-    //todo: Implement an endpoint to update ambassador
+    @DeleteMapping("/ambassadors/{id}")
+    public Ambassador deleteAmbassador(@PathVariable long id) throws ResourceNotFoundException {
+        if (ambassadorRepository.existsById(id)) {
+            Ambassador ambassador = ambassadorRepository.getOne(id);
+            ambassador.setStatus("REMOVED");
```

Comment:
Add status values to a constants class and refer from there instead of hardcoding